evennia/web/utils/general_context.py

```diff
@@ -10,17 +10,18 @@
 from evennia.utils.utils import get_evennia_version
 
 # Determine the site name and server version
-
-try:
-    GAME_NAME = settings.SERVERNAME.strip()
-except AttributeError:
-    GAME_NAME = "Evennia"
-SERVER_VERSION = get_evennia_version()
-try:
-    GAME_SLOGAN = settings.GAME_SLOGAN.strip()
-except AttributeError:
-    GAME_SLOGAN = SERVER_VERSION
-
+def set_game_name_and_slogan():
```

Comment:
I'm fine with wrapping things like this, but this is now a function (outside of the test suite) that needs a proper (albeit short) doc string as per the Evennia style guide. It might also be a good idea to add a `Notes:` section to this to point out that using a function here has a purpose for testing.